static/src/stylesheets/module/external/_from-content-api.scss

```diff
@@ -116,6 +116,11 @@
 .from-content-api pre {
```

Comment:
They don't know if it's used anywhere else. :disappointed: `pre` already had monospace font styling—I'm only adding a background, border, and padding here, so it shouldn't hurt anything.